src/main/java/me/ryanhamshire/GriefPrevention/EntityDamageHandler.java

```diff
@@ -236,13 +241,13 @@ private boolean isHostile(@NotNull Entity entity)
         if (type == EntityType.GHAST || type == EntityType.MAGMA_CUBE || type == EntityType.SHULKER)
             return true;
 
-        if (entity instanceof Slime slime) return slime.getSize() > 0;
-
-        if (entity instanceof Rabbit rabbit) return rabbit.getRabbitType() == Rabbit.Type.THE_KILLER_BUNNY;
+        if (entity instanceof Slime slime)
+            return slime.getSize() > 0 || slime.getTicksLived() < 1200;
```

Comment:
maybe add comment to better highlight this exception in handling "semi-tamables"